Lib9c/Action/BurnAsset.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections.Generic;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Exceptions;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Action
+{
+    [ActionType(TypeIdentifier)]
+    public class BurnAsset : ActionBase
+    {
+        private const int MemoMaxLength = 80;
+        public const string TypeIdentifier = "burn_asset";
+
+        private string _memo;
+
+        public BurnAsset()
+        {
+        }
+
+        public BurnAsset(Address owner, FungibleAssetValue amount, string memo)
+        {
+            Owner = owner;
+            Amount = amount;
+            Memo = memo;
+        }
+
+        public override IValue PlainValue =>
```

Comment:
How about move `PlainValue` to Above `LoadPlainValue` or move `LoadPlainValue` below `PlainValue`?
I think it's better to place deserialization/serialization codes in same place.

Reply:
I'll adjust them.

Reply:
Applied. https://github.com/planetarium/lib9c/pull/2190/commits/6d32a1144180fc60deae216bba0cb75e5553732f